apps/webClient/src/components/View/index.tsx

```diff
@@ -41,48 +112,64 @@ const useStyles = makeStyles(theme => ({
     flexGrow: 1,
   },
   redPageWrapper: {
-    backgroundColor: RED,
-    background: RED_GRADIENT,
+    backgroundColor: DARK_ORANGE,
+    backgroundImage: RED_GRADIENT,
```

Comment:
Its helpful to make a note why you switched from `background` to `backgroundImage` so that reviewers don't have to necessarily look it up